ionoscloud/resource_server.go

```diff
@@ -394,7 +401,7 @@ func resourceServer() *schema.Resource {
 									"type": {
 										Type:     schema.TypeString,
 										Optional: true,
-										Computed: true,
```

Comment:
@cristiGuranIonos why isn't `type` not computed anymore? Do we need to add a comment here?